docs/java/jvm/classloader.md

```diff
@@ -55,7 +55,7 @@ The GrandParent of ClassLodarDemo's ClassLoader is null
 `AppClassLoader`的父类加载器为`ExtClassLoader`，
 `ExtClassLoader`的父类加载器为 null，**null 并不代表`ExtClassLoader`没有父类加载器，而是 `BootstrapClassLoader`** 。
 
-其实这个双亲翻译的容易让别人误解，我们一般理解的双亲都是父母，这里的双亲更多地表达的是“父母这一辈”的人而已，并不是说真的有一个 Mother ClassLoader 和一个 Father ClassLoader 。另外，类加载器之间的“父子”关系也不是通过继承来体现的，是由“优先级”来决定。官方 API 文档对这部分的描述如下:
+其实这个双亲翻译的容易让别人误解，我们一般理解的双亲都是父母，这里的双亲更多地表达的是“父母这一辈”的人而已，并不是说真的有一个 Parent ClassLoader 和一个 Parent ClassLoader 。另外，类加载器之间的“父子”关系也不是通过继承来体现的，是由“优先级”来决定。官方 API 文档对这部分的描述如下:
```

Comment:
This part of the document explains why it is confusing to translate "parent" to 双亲, the only gender-neutral Chinese word for parent(s) -- 双亲 is explicitly plural, meaning that the word means "both parents". (Imagine there is no "parent" in English, only "both-parents".) The line reads:

> This translation of "both-parents" is misleading, because "both-parents" means "mother and father" for most of us. Here, the word both-parents is more like "the parents' generation". A "both-parents classloader" does not imply both a "mother classloader" and a "father classloader".

The proposed change is:

> A "both-parents classloader" does not imply both a "parent classloader" and a "parent classloader".

Therefore, changing "mother and father" to "parent and parent" makes no sense.